apps/docs/spec/supabase_py_v2.yml

```diff
@@ -6425,72 +6425,81 @@ functions:
         isSpotlight: true
         code: |
           ```python
-          channel = supabase.channel("room1")
+          channel = await supabase.channel("room1")
```

Comment:
From this snippet alone, it's not very clear where the `supabase` object comes from.

After a bit digging, it seems to be an `AClient` object?